### ISSUE TITLE
Locale strings should match the official abbreviations

### ISSUE THREAD

**Issue by stanislav-slavov** [aeternity/superhero-wallet#470]
We should change the locale strings to match its original representations. For example:

- Our locale strings: en, cn
- Official abbreviations: en-US, zh-CN

**Comment by davidyuk** [comment]
I agree regarding `cn` => `zn`, but can't we assume `en` a superset of `en-GB`, `en-US` and so? Of course, until we won't have any difference between these regions.
